README.id.md

```diff
@@ -0,0 +1,67 @@
+# IndoNLU 
+![Ditunggu Pull Requestsnya](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat) [![Lisensi Github](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/indobenchmark/indonlu/blob/master/LICENSE) [![Perjanjian Kontributor](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](code_of_conduct.md)
```

Comment:
Apparently, current `code_of_conduct.md` link is 404. please change code_of_conduct.md to https://github.com/indobenchmark/indonlu/blob/master/code_of_conduct.md. Nitip ubah ya. Thanks!

Reply:
Done!